libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/voice/MapboxSpeechPlayer.java

```diff
@@ -25,30 +25,34 @@
  * Will retrieve synthesized speech mp3s from Mapbox's API Voice.
  * </p>
  */
-class MapboxSpeechPlayer implements InstructionPlayer {
+class MapboxSpeechPlayer implements SpeechPlayer {
+
   private static final long TEN_MEGABYTE_CACHE_SIZE = 10 * 1098 * 1098;
   private static final String OKHTTP_INSTRUCTION_CACHE = "okhttp_instruction_cache";
   private static final String MAPBOX_INSTRUCTION_CACHE = "mapbox_instruction_cache";
   private static final String SSML_TEXT_TYPE = "ssml";
+  private static final String TEXT_TYPE = "text";
   private static final String ERROR_TEXT = "Unable to set data source for the media mediaPlayer! %s";
-  private Queue<File> instructionQueue;
+
   private VoiceInstructionLoader voiceInstructionLoader;
+  private SpeechAnnouncement announcement;
+  private SpeechListener speechListener;
   private MediaPlayer mediaPlayer;
-  private InstructionListener instructionListener;
-  private boolean isMuted;
+  private Queue<File> instructionQueue;
   private File mapboxCache;
   private Cache okhttpCache;
+  private boolean isMuted;
 
   /**
    * Construct an instance of {@link MapboxSpeechPlayer}
    *
-   * @param context to setup the caches
-   * @param language for which language
+   * @param context     to setup the caches
```

Comment:
we should be consistent with javadoc param spacing

Reply:
All of the javadoc spacing was moved to align to the length of the longest `@param`, in this case `accessToken`:
```
   * @param context     to setup the caches
   * @param language    for which language
   * @param accessToken a valid Mapbox access token
```
This is how I had always done it, but if we'd like to agree on a set style here I'm happy to conform / adjust cc @Guardiola31337

Reply:
That’s correct. But it’s true that we don’t have a checkstyle rule that enforces it. So if you reformat (as I do constantly :sweat_smile: ⌥⌘L) it gets formatted as @danesfeder mentioned.

Reply:
> We should add this to checkstyle, or at least going forward start to update non-conforming javadoc as we run into it.

Yeah as I mentioned we don't have one and it'd be awesome to add it. I briefly looked into it and it's a tricky one though (more difficult than I initially thought). I'll keep digging.